spec/factories/products.rb

```diff
@@ -7,6 +7,12 @@
     is_hidden false
     initial_order_status_id { FactoryGirl.create(:order_status, name: "New").id }
 
+    after(:build) do |product|
+      if product.facility_account.present?
```

Comment:
It's common in specs to set both a `facility` and `facility_account` when creating a `Product`. Having to explicitly set `facility` felt redundant, so I added this.